src/orbital-helpers/localize.h

```diff
@@ -44,14 +43,19 @@ namespace forte {
 class LOCALIZE {
   public:
     LOCALIZE(std::shared_ptr<Wavefunction> wfn, Options& options,
-             std::shared_ptr<ForteIntegrals> ints, std::shared_ptr<MOSpaceInfo> mo_space_info);
+             std::shared_ptr<ForteIntegrals> ints);
 
     ~LOCALIZE();
 
-    void split_localize();
 
-    void full_localize();
+    // Sets the orbitals to localize, bool to split localize
+    void set_orbital_space( std::vector<int>& orbital_spaces );
```

Comment:
This is great. Can we make it a bit more general so that it can accept a vector of vectors with the MO indices of the various groups of orbitals that we want to localize?

Reply:
I wasn't too sure how to have a vector of vectors as an option type, but I suppose the orbital_spaces_ object could be initialized from a vector of vectors. The orbital_spaces_ vector is a read as a set of index pairs, where the pair defines the first and last orbitals in a given set to be localized together. This does require a set to contain only orbitals with continuous indices, but that seems like a reasonable requirement.